src/Options.js

```diff
@@ -64,7 +64,9 @@ class Options {
     size: number;
     textSize: number;
     phantom: boolean;
-    fontFamily: string | void;
+    font: string;
+    oldTextFont: boolean;
+    fontFamily: string;
     fontWeight: string;
     fontShape: string;
```

Comment:
I was thinking we'd have properties like:
```
mathFont: string;
textFontFamily: string;
textFontWeight: string;
textFontShape: string;
mode: "text" | "math";
```
and then the API, would look something like:
```
.withMathFont(font: string);
.withTextFontFamily(family: string);
.withTextFontWeight(weight: string);
.withTextFontShape(shape: string);
```

Reply:
Hmm, how would the old text fonts be handled? I.e. `\rm \it`

Reply:
I forgot about `\rm \it`, but I think we could modify the suggested API in the following way:
```
.withTextFontFamily(family: string, isOldCommand: boolean = false);
.withTextFontWeight(weight: string, isOldCommand: boolean = false);
.withTextFontShape(shape: string, isOldCommand: boolean = false);
```

Reply:
I don't think `withTextFontFamily/Weight/Shape` would make sense with old font commands. Old font commands effectively change all 3, not just one. Additionally you'd lose the state when you exit the old font mode (i.e. `\textbf{{\rm hello} world}`). They behave much more similar to the math commands, which is why I opted to leave the naming conventions generic.

Reply:
`withTextFontFamily` would look something like:
```
withTextFontFamily(family: string, isOldCommand: boolean) {
   if (isOldCommand) {
       this.extend({
           fontFamily: fontFamily || this.fontFontFamily,
           fontWeight: "",
           fontStyle: "",
      });
   } else {
        this.extend({
            fontFamily: fontFamily || this.fontFamily,
        });
   }
};
```
As for the question of losing state, `{\rm hello}` creates a new `options` object when processing that group which shouldn't affect the `options` object for the `\textbf` group.

Reply:
Hmm, I'll give this more thought either later today or tomorrow. Thanks for the quick review.